kube-runtime/src/controller.rs

```diff
@@ -261,7 +360,7 @@ where
 ///     let cms = Api::<ConfigMap>::all(client.clone());
 ///     Controller::new(cmgs, ListParams::default())
 ///         .owns(cms, ListParams::default())
-///         .run(reconcile, error_policy, context)
+///         .run(reconcile, || backoff::backoff::Constant::new(Duration::from_secs(60)), context)
```

Comment:
Maybe we could have a .backoff builder on Controller here for readability?

Reply:
The downside there is that we either need to introduce a new generic or box the `Backoff`.

Reply:
Oh, does it get awkward if we try to do something like:

```rust
Controller::(..)
    .error(|| backoff::Constant::default())
    .run(reconcile, context)
```

I guess I wanted to have a default like what you have there without having to be explicit about the lambda.